toolchain/gcc-9.5.0-01-arm-phoenix-multilib.patch

```diff
@@ -0,0 +1,13 @@
+--- gcc-9.3.0-orig/gcc/config.gcc	2020-03-12 12:07:21.000000000 +0100
++++ gcc-9.3.0/gcc/config.gcc	2020-03-18 19:25:57.937836171 +0100
```

Comment:
Just a note that it have  old version in patch header. I know that you just copy 9.3.0 patches. 

It will be good as long as we do `patch -p1` while applying. But maybe it's worth considering to change it in the way that git doing that (a/path b/path) - not sure if it can be done with diffutils diff.

Reply:
Left on purpose to highlight that these patches were actually done for 9.3.0 (on 9.5.0 they do apply but with possibly-unexpected offset):
![image](https://github.com/phoenix-rtos/phoenix-rtos-build/assets/3939983/6ed35f3f-2167-4b4d-b8b0-0c3458970b0f)

Actually on host I've made them hardlinks to original patches, but it obviously didn't work in git. I've wanted to avoid symlinks as I don't know how it would work on windows.